src/detectors/nosql_injection/nosql_injection.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=nosql-injection@v1.0 defects=1}
+var AWS = require('aws-sdk')
+var express = require("express")
+var app = express()
+function noSqlInjectionNoncompliant() {
+    app.get(path, function(req,res) {
```

Comment:
Should it be `/api/getallusers` to keep it consistent?

Reply:
Updated